external_status_checks.go

```diff
@@ -88,3 +88,99 @@ func (s *ExternalStatusChecksService) ListProjectStatusChecks(pid interface{}, o
 
 	return pscs, resp, err
 }
+
+type SetExternalStatusCheckStatusOptions struct {
+	SHA                   *string `url:"sha" json:"sha"`
+	ExternalStatusCheckID *int    `url:"external_status_check_id" json:"external_status_check_id"`
+	Status                *string `url:"status,omitempty" json:"status,omitempty"`
+}
+
+// SetExternalStatusCheckStatus set status of an external status check
+//
+// Gitlab API docs:
+// https://docs.gitlab.com/ee/api/status_checks.html#set-status-of-an-external-status-check
+func (s *ExternalStatusChecksService) SetExternalStatusCheckStatus(pid interface{}, mergeRequestIID int, opt *SetExternalStatusCheckStatusOptions, options ...RequestOptionFunc) (*Response, error) {
```

Comment:
Following the order in the docs, this method should be defined below `ListMergeStatusChecks`